src/auth/auth.controller.ts

```diff
@@ -41,4 +55,28 @@ export class AuthController {
   signIn(@Body() loginCredentialsDto: LoginCredentialsDto): Promise<Tokens> {
     return this.authService.signIn(loginCredentialsDto);
   }
+
+  @Post('forgot-password')
+  @HttpCode(200)
```

Comment:
i'm thinking if `forgot-password` should also return 204 by default if `reset-password` returns 204